tests/apollo/test_skvbc_auto_view_change.py

```diff
@@ -129,4 +127,4 @@ async def test_auto_vc_all_nodes_up_fast_path(self, bft_network, tracker):
         await skvbc.assert_kv_write_executed(key, val)
         await bft_network.assert_fast_path_prevalent()
 
-        await tracker.tracked_read_your_writes()
+        await skvbc.read_your_writes(self)
```

Comment:
why do we need to pass the test class instance as a parameter ?

Reply:
I didn't change the original code for read_your_writes which uses test_class.assertEqual.

Reply:
why can't we just use `assert` instead of `test_class.assertEqual` ?